### DISCUSSION TITLE
Cannot reach CKAN through port 5000 in browser

### DISCUSSION THREAD

**Discussion by GitYepper** [ckan/ckan#6083] [Q&A] [answered]
**CKAN version**
2.9.2 

**Describe the bug**
I am not able to view the CKAN panel through port 5000 in the browser ('Unable to connect') after installation from source - even though it says

>  _Running CKAN on http://localhost:5000_ 

on the terminal. 

**Steps to reproduce**

Followed all the steps on:
https://docs.ckan.org/en/2.9/maintaining/installing/install-from-source.html

Everthing worked well and the CKAN instance even seems to run on port 5000. 
I already spent some time to find the issue and I think it has to do with the ckan.site_url in the ckan.ini. I already tried various URLs/IPs (demo.ckan.org, localhost and also my servers IP) but none of them worked. 

 Currently I have the following line in my ckan.ini file:

> _ckan.site_url = http://127.0.0.1:5000_

As the whole thing is running on an external server I tried to reach the panel over the server IP. I also have 127.0.0.1 as my solr_url which is reachable over the server ip (and the corresponding port 8983) so I thought this might also work here, but nope..

**Expected behavior**
I want to view the CKAN panel throuh port 5000 via the browser.

**Additional details**
As mentioned, I think the issue is the ckan.site_url in the ckan.ini where I am not really sure what is the correct URL when I want to reach the CKAN panel over an external IP.

**Comment by kowh-ai** [comment, marked as answer]
You could try the `--host` and `--port` options:  https://docs.ckan.org/en/latest/maintaining/cli.html?highlight=--host#run-start-development-server

**Reply by GitYepper** [reply]
Thank you! It is working now. I can now reach the CKAN panel over my server IP through port 5000 after running the following command:

`ckan run --host SERVER_IP --port 5000`

**Comment by anuveyatsu** [comment]
Looks similar to https://github.com/ckan/ckan/discussions/6025
